### PR TITLE
Swedish translation

### PR DESCRIPTION
**Pull request by christiannageby**
Added a swedish translation for the plugin.

### CODE REVIEW

File: skins/larry/lastlogin.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .lastlogin TABLE {
 	border-collapse:collapse;
-	border:1px solid #bbd3da;
+	border:1px solid #1b1b1b;
```
Comment (dsoares) [inline skins/larry/lastlogin.css]: Hi,
Why did you change the border color? Any specific reason?

**Comment by dsoares** [comment]
Hi,

Thanks for your contribution. I commented on the change of the border color; when you answer, i'll approve the PR.

**Comment by dsoares** [comment]
Hi,
12 days are gone and you didn't answered my question.
I don't agree with the change of the border color.  Please, do a PR without that change. All the other commits are OK to me.

**Comment by dsoares** [comment]
Since you didn't give me any feedback on my request about the CSS changes, i only merged your translation and implemented your idea for masking the IP. See changelog of release 1.1.0.

File: skins/larry/lastlogin.css (hunk 1 above)
Comment (christiannageby) [reply skins/larry/lastlogin.css]: Oh, must have done it by mistake, That was for another project of mine.
